sem_init(3) is part of libc on FreeBSD, so we ended up not linking against
libpthread. Look for a symbol that causes us to load it.
libtesseract.so is not linked against libomp if openmp is enabled,
which makes dependent ports fail. This adds a workaround by linking
directly against libomp.
--- configure.ac.orig	2021-11-30 17:48:16 UTC
+++ configure.ac
@@ -442,6 +442,9 @@ esac
 # ----------------------------------------
 
 AC_SEARCH_LIBS([pthread_create], [pthread])
+if test "$enable_openmp" != no; then
+  AC_SEARCH_LIBS([omp_get_thread_num ], [omp])
+fi
 
 
 # ----------------------------------------
